diff -Nru pcre3-8.35/debian/libpcrecpp0v5.symbols pcre3-8.35/debian/libpcrecpp0v5.symbols
--- pcre3-8.35/debian/libpcrecpp0v5.symbols	2015-08-04 18:22:05.000000000 +0000
+++ pcre3-8.35/debian/libpcrecpp0v5.symbols	2015-08-08 08:53:34.000000000 +0000
@@ -1,4 +1,5 @@
 libpcrecpp.so.0 libpcrecpp0v5 #MINVER#
+ _ZN7pcrecpp2RE9QuoteMetaERKNS_11StringPieceE@Base 2:8.35-7.1
  (c++)"pcrecpp::Arg::parse_char(char const*, int, void*)@Base" 7.7
  (c++)"pcrecpp::Arg::parse_double(char const*, int, void*)@Base" 7.7
  (c++)"pcrecpp::Arg::parse_float(char const*, int, void*)@Base" 7.7
@@ -61,7 +62,7 @@
  (c++)"pcrecpp::RE::no_arg@Base" 7.7
  (c++)"pcrecpp::RE::NumberOfCapturingGroups() const@Base" 7.7
  (c++)"pcrecpp::RE::PartialMatch(pcrecpp::StringPiece const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&, pcrecpp::Arg const&) const@Base" 7.7
- (c++)"pcrecpp::RE::QuoteMeta[abi:cxx11](pcrecpp::StringPiece const&)@Base" 7.7
+#MISSING: 2:8.35-7.1# (c++)"pcrecpp::RE::QuoteMeta[abi:cxx11](pcrecpp::StringPiece const&)@Base" 7.7
  (c++)"pcrecpp::RE::~RE()@Base" 7.7
  (c++)"pcrecpp::RE::~RE()@Base" 7.7
  (c++)"pcrecpp::RE::Replace(pcrecpp::StringPiece const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*) const@Base" 7.7
