diff -u freetype-2.5.2/debian/patches-ft2demos/series freetype-2.5.2/debian/patches-ft2demos/series
--- freetype-2.5.2/debian/patches-ft2demos/series
+++ freetype-2.5.2/debian/patches-ft2demos/series
@@ -3,0 +4 @@
+ftcommon-null-pointer.patch
diff -u freetype-2.5.2/debian/rules freetype-2.5.2/debian/rules
--- freetype-2.5.2/debian/rules
+++ freetype-2.5.2/debian/rules
@@ -18,6 +18,10 @@
 endif
 export DEB_CFLAGS_MAINT_APPEND += $(shell dpkg-buildflags --get CPPFLAGS)
 
+ifeq ($(shell $(CC) --version | grep -c "clang version"),1)
+DEB_CFLAGS_MAINT_APPEND += -Wno-error=long-long -Wno-error=extended-offsetof -Wno-error=self-assign
+endif
+
 # always abort if there are any problems with the symbols file
 DPKG_GENSYMBOLS_CHECK_LEVEL = 4
 export DPKG_GENSYMBOLS_CHECK_LEVEL
only in patch2:
unchanged:
--- freetype-2.5.2.orig/debian/patches-ft2demos/ftcommon-null-pointer.patch
+++ freetype-2.5.2/debian/patches-ft2demos/ftcommon-null-pointer.patch
@@ -0,0 +1,13 @@
+Index: ft2demos-2.5.1.orig/src/ftcommon.c
+===================================================================
+--- ft2demos-2.5.1.orig/src/ftcommon.c
++++ ft2demos-2.5.1/src/ftcommon.c
+@@ -242,7 +242,7 @@
+         if ( has_extension )
+           memcpy( suffix, orig, 5 );
+         else
+-          suffix = '\0';
++          *suffix = '\0';
+       }
+ 
+       if ( (*aface)->charmaps )
